### PR TITLE
Added Docker images for Debezium 0.3 release and tutorial

### PR DESCRIPTION
**Pull request by rhauch**
Updates the Docker images to add images with 0.3 labels.

### CODE REVIEW

File: kafka/0.3/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,55 @@
+FROM debezium/jdk8:8u92
+
+MAINTAINER Debezium Community
+
+#
+# Set the version, home directory, and MD5 hash.
+# MD5 hash taken from http://kafka.apache.org/downloads.html for this version of Kafka
+#
+ENV KAFKA_VERSION=0.10.0.0 \
```
Comment (mvj3) [inline kafka/0.3/Dockerfile]: Great! Our team is waiting for 0.10.0.0 version of Kafka

**Comment by hchiorean** [comment]
@rhauch does this have the same connection issues with kafka 0.10.0.1 ?

**Comment by rhauch** [comment]
@hchiorean, I haven't tested the images with 0.10.0.1 yet. Will be doing that today.

**Comment by hchiorean** [comment]
@rhauch: I tried building the latest images locally, but the following URL `https://oss.sonatype.org/content/repositories/iodebezium-1007` doesn't exist (and is referenced atm from `connect/0.3`)

**Comment by rhauch** [comment]
@hchiorean, yes, I know. It doesn't build for me but this morning I'm going to try another release. Local builds using local plugin archives works after I corrected the port default. I'll update this PR when the official archives are in Maven Central.

**Pull request merged**